selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/assistant/CheckFindActionFeatureInCheTest.java

```diff
@@ -91,6 +94,7 @@
   @Inject private Menu menu;
   @Inject private Ide ide;
   @Inject private TestProjectServiceClient testProjectServiceClient;
+  @Inject private SeleniumWebDriver seleniumWebDriver;
```

Comment:
What is the propose of injection of **seleniumWebDriver**? It looks redundant here.